Pages/Players/PlayerListPage.cs

```diff
@@ -11,6 +12,14 @@ namespace LineUp.Pages.Players;
 public class PlayerListPageState
 {
     public IEnumerable<Player> Players { get; set; } = [];
+
+    public Dictionary<Position, List<Player>> GetGroupedPlayers()
```

Comment:
The Collection View supports [grouping](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/collectionview/grouping?view=net-maui-8.0) I believe